include/zephyr/bluetooth/gatt.h

```diff
@@ -72,6 +72,30 @@ enum {
 	 *  passed to write callback.
 	 */
 	BT_GATT_PERM_PREPARE_WRITE = BIT(6),
+
+	/** @brief Attribute write permission with 128bit entropy key connection.
+	 *
+	 *  If set, write access requires connection using an at least 128bit entropy
+	 *  key derived from one of the following sources:
+	 *  - LE Secure Connections
+	 *  - BR/EDR Secure Connections
+	 *  - OOB method
+	 *
+	 *  This is a security requirement on e.g. Basic Audio Profile (v1.0, Section 9.1)
+	 */
+	BT_GATT_PERM_READ_128B_ENTROPY = BIT(7),
+
+	/** @brief Attribute write permission with 128bit entropy key connection.
+	 *
+	 *  If set, write access requires connection using an at least 128bit entropy
+	 *  key derived from one of the following sources:
+	 *  - LE Secure Connections
+	 *  - BR/EDR Secure Connections
+	 *  - OOB method
+	 *
+	 *  This is a security requirement on e.g. Basic Audio Profile (v1.0, Section 9.1)
+	 */
+	BT_GATT_PERM_WRITE_128B_ENTROPY = BIT(8),
```

Comment:
```suggestion
	/** @brief Attribute write permission with 128-bit entropy key connection.
	 *
	 *  If set, write access requires connection using an at least 128-bit entropy
	 *  key derived from one of the following sources:
	 *  - LE Secure Connections
	 *  - OOB method
	 *
	 */
	BT_GATT_PERM_READ_128B_ENTROPY = BIT(7),

	/** @brief Attribute write permission with 128-bit entropy key connection.
	 *
	 *  If set, write access requires connection using an at least 128-bit entropy
	 *  key derived from one of the following sources:
	 *  - LE Secure Connections
	 *  - OOB method
	 *
	 */
	BT_GATT_PERM_WRITE_128B_ENTROPY = BIT(8),
```

I don't see any reason to mention the BR/EDR part if we do not support it. 

I also feel like GATT should not reference higher layer profiles like BAP as the reasoning for this (but this one I am OK with if it stays). 

Is OOB method always 128-bit entropy btw? I don't know.